### PR TITLE
Added Fatal System Error from Royal Opera House

### PR DESCRIPTION
**Pull request by yaustar**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,6 +162,10 @@
 
 - [Party Space](https://www.party.space/) - Virtual events platform.
 
+## Experiences
+
+- [Fatal System Error](https://playcanv.as/p/A5qC9ftM/?overlay=false) - An interactive web-based opera experience created by Dumbworld from the Royal Opera House.
```
Comment (willeastcott) [inline README.md:167]: This really belongs in the Advertising section.

Comment (yaustar) [reply README.md:167]: It's not really an advert for a product though? https://www.roh.org.uk/tickets-and-events/festival/8bit-details

Comment (willeastcott) [reply README.md:167]: Under advertising, I normally include: playable ads and branded experiences. This is kinda the latter.

File: README.md [hunk 2]
```diff
@@ -91,6 +91,7 @@
 - [DOOH Invaders](https://vimeo.com/136313614) - World's largest Space Invaders clone unveiled at Cannes Lions.
 - [Ride Along 2](https://vimeo.com/152996271) - Using WebGL to promote a Hollywood movie release.
 - [Rugby World Cup](https://www.youtube.com/watch?v=onOJBCC8R7A) - Promoting the Rugby World Cup in London's Piccadilly Circus.
+- [Fatal System Error](https://playcanv.as/p/A5qC9ftM/?overlay=false) - An interactive web-based opera experience created by Dumbworld from the Royal Opera House.
```
Comment (willeastcott) [inline README.md:94]: You're gonna hate me - alphabetical order. :wink:

**Review by willeastcott — APPROVED** [review]

**Comment by yaustar** [comment]
I should read the contribution guidelines 😅

**Pull request merged**